src/types/global.d.ts

```diff
@@ -41,28 +39,24 @@ declare type SortOption =
     | 'Price: lowest to highest'
     | 'Price: highest to lowest';
 
+declare interface StockOption {
+    qtyInStock: number;
+    }    
 
-type Type = string;
-
-//mock data type,  this will be defined as an interface for Backend Data
-declare type Brand =string;
-
-//mock data type
-declare type PriceRange = string;
-
-//mock data type,  this will be defined as an interface for Backend Data
-declare type Color =  string;
-
-//mock data type,  this will be defined as an interface for Backend Data
-declare type Stock = string;
-
+declare interface AvailableOptions {
+    types: string[];
+    brands: string[];
+    priceRanges: string[];
+    colors: string[];
```

Comment:
I'm unsure whether simplifying types of each filter to string array is a good ida or not. It does make the code simpler and easier to understand, but potentially reduces type safety.